### PR TITLE
Refactor to new APIs

### PR DESCRIPTION
**Pull request by darko-mijic**
This PR updates Todos to all the latest APIs and it fixes a bug introduced in space:base@4.0.0.

### CODE REVIEW

**Comment by darko-mijic** [comment]
Can you review this quickly @rhyslbw? If this is the best way to use new APIs Donations will be updated using this as a template.

File: packages/domain/package.js [hunk 1]
```diff
@@ -35,8 +35,9 @@ Package.onTest(function(api) {
   api.use([
     'mongo',
     'underscore',
-    'space:testing@2.0.0',
-    'space:testing-event-sourcing@0.1.0',
+    'space:testing@3.0.1',
+    'space:testing-event-sourcing@3.0.0',
+    'space:testing-event-sourcing@3.0.0',
```
Comment (rhyslbw) [inline packages/domain/package.js]: Duplicate line

File: packages/domain/source/server/todos/todo-list.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 Space.eventSourcing.Aggregate.extend('Todos.TodoList', {
 
-  fields: {
-    name: String,
-    todos: [Todos.TodoItem]
+  fields() {
```
Comment (rhyslbw) [inline packages/domain/source/server/todos/todo-list.js]: Should revert this. see https://github.com/meteor-space/event-sourcing/pull/51#issuecomment-174206412

Comment (darko-mijic) [inline packages/domain/source/server/todos/todo-list.js]: Yes, after the #51 is merged. I will maintain this to always work with current APIs. Lets leave this PR open until #51 is merged.

Comment (rhyslbw) [inline packages/domain/source/server/todos/todo-list.js]: Pretty sure we're not going to merge that PR, as this particular API probably won't change

**Comment by darko-mijic** [comment]
@DominikGuzei: can you help with failing tests here? There seams to be a bug in testing-event-sourcing.

**Comment by rhyslbw** [comment]
How are we going with this guys? Would be great to get this merged for anyone checking out the new versions

**Comment by darko-mijic** [comment]
This is ready for merging.

**Pull request merged**